dataservice/util/data_gen/data_generator.py

```diff
@@ -179,6 +180,14 @@ def _outcomes_choices(self):
         self.vital_status_list = ['Alive', 'Dead', 'Not Reported']
         self.disease_related_list = [True, False, 'Not Reported']
 
+    def _get_unique_sites(self, diagnoses):
```

Comment:
Probably no need for "unique" here unless you really do intend to return the set of anatomical sites